tests/test_other.py

```diff
@@ -10645,3 +10645,6 @@ def test_standard_library_mapping(self):
     self.assertContained(' -lc-mt ', err)
     self.assertContained(' -ldlmalloc-mt ', err)
     self.assertContained(' -lcompiler_rt-mt ', err)
+
+  def test_explict_gl_linking(self):
+    self.run_process([EMCC, test_file('other/test_explict_gl_linking.c'), '-sNO_AUTO_NATIVE_LIBRARIES', '-lGL'])
```

Comment:
Maybe add a comment here? I think I understand from your explanation what this checks now, but just from the test code it's not obvious. In particular, what does "explicit" mean in the title - ? The comment could explain that.

Reply:
Done.